governator-core/src/main/java/com/netflix/governator/ProvisionDebugModule.java

```diff
@@ -6,16 +6,16 @@
  * Install this module to log a Provision report after the Injector is created.
  */
 public final class ProvisionDebugModule extends SingletonModule {
```

Comment:
seems more like a ProvisionReportModule if that's what it generates.  Seems like debugging is an unrelated or second-order concept